### ISSUE TITLE
Advice needed: Visualizing realtime incoming data through native libs

### ISSUE THREAD

**Issue by kakyoism** [flutter/flutter#37374]
I need to visualize realtime time-series data coming from network at a very low latency (<30ms). The data transfer logic is in C++. User can control the visual via UI in realtime. As I understand, the implementation must go through the Platform Channel, which involves Dart <=> Swift/Kotlin <=> C++ dual-way communication. This sounds very expensive. I can't find success stories close to my specific need besides trivial attempts like these business charts: https://medium.com/flutter/tagged/data-visualization

**In short, would Flutter pull off a low-latency time-series data visualization task involving the usage of Platform Channel?**

So any suggestions would be appreciated.

**Comment by kf6gpe** [comment]
Thanks for using Flutter! This is a support request, and it's better to engage the community with these --- please reach out to one of the forums at https://flutter.dev/support. I'm closing this issue as it appears not to be an issue with Flutter itself.
